ui/app/styles/core/forms.scss

```diff
@@ -282,17 +175,62 @@
   }
 }
 
+// File css
+.file {
+  user-select: none;
+}
+
+.file-name {
+  box-shadow: 0 4px 1px rgba(10, 10, 10, 0.06) inset;
+  border: 1px solid $ui-gray-300;
+  border-radius: 2px;
+  border-width: 1px 1px 1px 0 !important;
```

Comment:
Yep, sure do. Right now, if you see !important it's because it comes from Bulma. Just doing the copy/paste with very little changes from bulma styling and then I'll do a sweep through of all files to clean up importants, and variables and unused styles.